### PR TITLE
Module Documentation

### PR DESCRIPTION
**Pull request by kenlacroix**
Create documentation.

### CODE REVIEW

**Comment by kenlacroix** [comment]
Ready for review.

File: documentation/modules/post/windows/gather/dumplinks.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+                  SW_RESTORE.
+        Target file's MAC Times stored in lnk file:
+                  Creation Time     = 2019-12-11 23:44:30 -0700. (UTC)
+                  Modification Time = 2019-12-11 23:44:30 -0700. (UTC)
+                  Access Time       = 2019-12-11 23:44:30 -0700. (UTC)
+        Shortcut file is on a local volume.
+                  Volume Name =
+                  Volume Type = Fixed
+                  Volume SN   = 0x548EF20B
+        Target path = C:\Users\TEST\Desktop\myPasswords.txt&..\..\..\..\..\Desktop\myPasswords.txtC:\Users\TEST\Desktop(
```
Comment (h00die) [inline documentation/modules/post/windows/gather/dumplinks.md]: is this line cut off, or supposed to be two lines?  The `C:\Users\TEST\Desktop(` at the end seems out of place

File: documentation/modules/post/windows/gather/dumplinks.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+## Options
+
+  **SESSION**
+
+  The session to run the module on.
+
+
+## Scenarios
+
```
Comment (h00die) [inline documentation/modules/post/windows/gather/dumplinks.md]: OS?

File: documentation/modules/post/windows/gather/enum_devices.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+## Options
+
+  **SESSION**
+
+  The session to run the module on.
+
+
+## Scenarios
+
```
Comment (h00die) [inline documentation/modules/post/windows/gather/enum_devices.md]: OS?

File: documentation/modules/post/windows/gather/enum_logged_on_users.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+  2. Get meterpreter session
+  3. Do: ```use post/windows/gather/enum_logged_on_users```
+  4. Do: ```set SESSION <session id>```
+  5. Do: ```run```
+
+## Options
+
+  **CURRENT**
+
+  Enumerate currently logged on users.
```
Comment (h00die) [inline documentation/modules/post/windows/gather/enum_logged_on_users.md]: default?

File: documentation/modules/post/windows/gather/enum_logged_on_users.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+
+  **RECENT**
+
+  Enumerate Recently logged on users.
```
Comment (h00die) [inline documentation/modules/post/windows/gather/enum_logged_on_users.md]: default?

File: documentation/modules/post/windows/gather/enum_logged_on_users.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+
+  **SESSION**
+
+  The session to run this module on.
+
+## Scenarios
+
```
Comment (h00die) [inline documentation/modules/post/windows/gather/enum_logged_on_users.md]: OS?

File: documentation/modules/post/windows/gather/screen_spy.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
+## Vulnerable Application
+
+  This Module lists current TCP sessions.
```
Comment (h00die) [inline documentation/modules/post/windows/gather/screen_spy.md]: copy pasta?

File: documentation/modules/post/windows/gather/screen_spy.md [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+  5. Do: `run`
+
+## Options
+
+  **SESSION**
+
+  The session to run the module on.
+
+## Scenarios
+
```
Comment (h00die) [inline documentation/modules/post/windows/gather/screen_spy.md]: OS?

**Review by h00die — CHANGES_REQUESTED** [review]

**Comment by h00die** [comment]
Docs are looking pretty good, just a few quick minor things

File: documentation/modules/post/windows/gather/screen_spy.md (hunk 7 above)
Comment (kenlacroix) [reply documentation/modules/post/windows/gather/screen_spy.md]: yes

File: documentation/modules/post/windows/gather/dumplinks.md (hunk 1 above)
Comment (kenlacroix) [reply documentation/modules/post/windows/gather/dumplinks.md]: I thought so to, but that is the exact output.

**Comment by h00die** [comment]
Thanks for the additional docs @kenlacroix !

**Comment by h00die** [comment]
# Release Notes
This PR adds more module docs, mainly for windows post modules.

**Pull request merged**